Dockerfile

```diff
@@ -4,41 +4,22 @@ FROM python:3.8-alpine
 ENV USER="app"
 ENV GROUP="app"
 ENV HOME="/home/$USER"
-ENV APP_HOME="$HOME/web"
+ENV PROJECT_HOME="$HOME/web"
 ENV PATH="$HOME/env/bin:$PATH"
 ENV VIRTUAL_ENV="$HOME/env"
 
 # env - python tweeking
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
 
-# env - app
-ENV DEBUG=1
-ENV DJANGO_SUPERUSER_USERNAME="juntagrico"
-ENV DJANGO_SUPERUSER_PASSWORD="juntagrico"
-ENV DJANGO_SUPERUSER_EMAIL="juntagrico@localhost.localhost"
-ENV JUNTAGRICO_SECRET_KEY="juntagrico"
-ENV JUNTAGRICO_DATABASE_BACKEND="django.db.backends.sqlite3"
-ENV JUNTAGRICO_DATABASE_NAME="juntagrico.sqlite3"
-ENV JUNTAGRICO_DATABASE_USER=
-ENV JUNTAGRICO_DATABASE_PASS=
-ENV JUNTAGRICO_DATABASE_HOST=
-ENV JUNTAGRICO_DATABASE_PORT=
-ENV JUNTAGRICO_EMAIL_HOST="localhost"
-ENV JUNTAGRICO_EMAIL_USER="juntagrico@localhost.localhost"
-ENV JUNTAGRICO_EMAIL_PASS="secret"
-ENV JUNTAGRICO_EMAIL_PORT=587
-ENV JUNTAGRICO_EMAIL_TLS="true"
-ENV JUNTAGRICO_FQDN="localhost"
```

Comment:
We should set a default (e.g. localhost) here: https://github.com/Rote-Beete/juntagrico-docker/blob/main/settings.py#L6

Reply:
For development, it is not really necessary to set `ALLOWED_HOSTS`. Looking at the [documentation](https://docs.djangoproject.com/en/3.1/ref/settings/#allowed-hosts)

> When DEBUG is True and ALLOWED_HOSTS is empty, the host is validated against ['.localhost', '127.0.0.1', '[::1]'].